botorch/acquisition/monte_carlo.py

```diff
@@ -212,7 +225,11 @@ def __init__(
                 before instantiating the acquisition function.
         """
         super().__init__(
-            model=model, sampler=sampler, objective=objective, X_pending=X_pending
+            model=model,
+            sampler=sampler,
+            objective=objective,
+            posterior_transform=posterior_transform,
+            X_pending=X_pending,
         )
         if prune_baseline:
             X_baseline = prune_inferior_points(
```

Comment:
Need to pass `posterior_transform` to `prune_inferior_points`

Reply:
good catch